### DISCUSSION TITLE
wont block ads

### DISCUSSION THREAD

**Discussion by TheTechRobo** [dhowe/AdNauseam#1875] [Q&A] [answered]
No ads are blocked.  i can still see them. also, the number doesn't change..

http://lab-lamp.scm.cityu.edu.hk/adntest/simple.html here the ad is still there and not blocked, plus the number doesn't change

**Comment by dhowe** [comment]
Hi @TheTechRobo this sounds to me like it may be a corrupt Firefox profile. Would you mind [creating a new profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles) and testing AdNauseam with it ?

**Reply by TheTechRobo** [reply]
Oh no, I forgot to say that I'm using Chrome.

Also, it does seem that I have a bunch of ads in the ad vault, but it's almost none (31). Plus, on most websites, it has the number 0, on ublock it had at LEAST a few per site. :confused:

**Comment by dhowe** [comment]
Its important to note that AdNauseam shows the number of (complete) ads that it finds, while uBlock shows the number of resources that it blocks or hides, which is generally a much higher number. So this is normal. But assuming AdNauseam is fully enabled (with hiding, clicking, and blocking malware on, as shown below), then you should NOT be seeing any ads. Can you double-check that your settings look like the ones below?

![image](https://user-images.githubusercontent.com/737638/122446774-08c03380-cf71-11eb-970d-eea041adfab4.png)

**Reply by TheTechRobo** [reply]
yeah, they look like that.

what is another good site for testing ads

**Comment by dhowe** [comment, marked as answer]
Can you try this one: http://lab-lamp.scm.cityu.edu.hk/adntest/simple.html

**Reply by TheTechRobo** [reply]
Sorry, I think I thought that that image shouldn't have been there. Sorry for bugging you. I think ti works

![image](https://user-images.githubusercontent.com/52163910/122498599-667a6d00-cfbd-11eb-9be0-c20d4d60f00b.png)

Update: After refresh the image isn'T even there. Again, sorry for wasting your time.
